### ISSUE TITLE
Dynamically imported component is rendered twice on the client

### ISSUE THREAD

**Issue by OndrejTuma** [vercel/next.js#10028]
# Bug report
When using dynamically imported component with `next/dynamic` function, my component is always rendered twice

## Describe the bug
![image](https://user-images.githubusercontent.com/22159416/72154276-89f4bd80-33b0-11ea-8e4a-44f7ff744c31.png)

This bug is quite self-descriptive

## To Reproduce
It can be seen in official repo in example with-dynamic-import

1. Go to https://github.com/zeit/next.js/tree/canary/examples/with-dynamic-import and clone it, or
2. check it in codesandbox https://codesandbox.io/s/nexjs-dynamic-import-double-render-g060p

## Expected behavior

Component should be rendered only once

## System information

- OS: macOS, Ubuntu
- Browser: Chrome
- Version of Next.js: 9.1.7

**Comment by tardnuk** [comment]
Just to clarify - it is not renderd twice on page, but it is rerenderd.

**Comment by BhupenT** [comment]
I am also facing this issue. @Rootre have you found any solution round this?

**Comment by Jekins** [comment]
I am also facing this issue.

**Comment by PaulinaQuintero** [comment]
In the sandbox I think it is rendered once 
![image](https://user-images.githubusercontent.com/63326136/85491341-f9461e80-b598-11ea-8ba6-248dc20610b6.png)

**Comment by balazsorban44** [comment]
Given the reproduction, this no longer reproduces. Please try upgrading to `next@latest`

**Comment by balazsorban44** [comment]
This issue has been automatically locked due to no recent activity. If you are running into a similar issue, please create a new issue with the steps to reproduce. Thank you.
